Tests/Core/Logging/FramesEventLoggerTests.swift

```diff
@@ -19,7 +19,7 @@ final class FramesEventLoggerTests: XCTestCase {
         stubCheckoutEventLogger = StubCheckoutEventLogger()
         stubDateProvider = StubDateProvider()
         subject = FramesEventLogger(
-            correlationID: stubcorrelationID,
+            getCorrelationID: { [stubcorrelationID] in stubcorrelationID },
```

Comment:
Would this alternative compile `getCorrelationID: { $0 },`?

I got stuck few seconds looking at that syntax, thinking how that comes as an array and returns an array when it wanted a String. 😕